Library/Styles/ButtonStyles.swift

```diff
@@ -19,6 +19,17 @@ public let baseButtonStyle =
   <> UIButton.lens.adjustsImageWhenDisabled .~ false
   <> UIButton.lens.adjustsImageWhenHighlighted .~ false
 
+// This is a temporary style meant to replace baseButtonStyle in the very near future
+public let baseNewButtonStyle: ButtonStyle = { button in
+  button
+    |> roundedButtonStyle
+    |> UIButton.lens.titleLabel.font .~ UIFont.ksr_callout().bolded
+    |> (UIButton.lens.titleLabel .. UILabel.lens.textAlignment) .~ NSTextAlignment.center
+    |> UIButton.lens.titleEdgeInsets .~ .init(all: Styles.grid(2))
+    |> UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode .~ NSLineBreakMode.byTruncatingMiddle
```

Comment:
Buttons will always by default truncate in the middle when the text is too long. cc @dannyalright

Reply:
A string like `Pledge US$ 124 or more` would be truncated to something like `Pledge US... more`. Can we have an exception for buttons like that? Ones that have the core information in the centre of the string?

I agree with middle-of-the-string truncation otherwise.

Reply:
Yep we can have an exception for the certain buttons. I'll update the truncation on the pledge button to be `.byTruncatingTail` 👍